configs/components/rubygem-facter.rb

```diff
@@ -1,6 +1,13 @@
 component 'rubygem-facter' do |pkg, settings, platform|
-  pkg.version '4.2.11'
-  pkg.md5sum '5a387c10f5917d88b46cacd5da2de6f1'
+  pkg.environment "GEM_HOME", settings[:gem_home]
+  pkg.environment "GEM_PATH", settings[:gem_home]
+  pkg.url("https://nightlies.puppet.com/downloads/gems/facter-nightly/facter-latest.gem")
```

Comment:
Ooohhh, so we'll need to wait until.... tomorrow and then point this at the versions we release.